build.gradle

```diff
@@ -129,7 +129,7 @@ tasks.withType(Test).configureEach {
     }
 }
 
-tasks.named("test") {
+tasks.named("test").configure {
```

Comment:
This shouldn't actually have an effect.  `tasks.named(...) { }` and `tasks.named(...).configure { }` do the same thing.  See https://docs.gradle.org/current/dsl/org.gradle.api.tasks.TaskContainer.html#org.gradle.api.tasks.TaskContainer:named(java.lang.String,%20org.gradle.api.Action).